src/data/Terra/TerraAssets.ts

```diff
@@ -100,7 +100,7 @@ const sortWhitelistCW20 = (data: CW20Whitelist) => {
 
   return fromPairs(
     sorted.map(([t, { decimals, ...item }]) => {
-      return [t, { ...item, decimals: decimals ?? 6 }]
```

Comment:
Would there ever be a case where decimals may be undefined and this would result in an error?